lib/vdsm/tool/config_lvm_filter.py

```diff
@@ -133,6 +133,11 @@ def config_with_devices(args, mounts, current_wwids, wanted_wwids):
     # Find VGs of mounted devices.
     vgs = {mnt.vg_name for mnt in mounts}
 
+    # Print config summary for the user.
+
+    _print_devices_info(vgs)
+    _print_summary(mounts, None, None, wanted_wwids)
```

Comment:
I see why we print the vgs before the mounts - this should be printed in the summary.
We can pass the vgs to the summary function, and it wil print the list in the right place.